.cirrus.yml

```
@@ -13,21 +20,33 @@ container_definition: &CONTAINER_DEFINITION
   namespace: default
   use_in_memory_disk: true
 
+setup_sonar_scanner: &SETUP_SONAR_SCANNER
+  setup_sonar_scanner_script:
+    - apt update -y && apt upgrade -y && apt update -y && apt install -y unzip       
+    - curl --create-dirs -sSLo $HOME/.sonar/sonar-scanner.zip https://binaries.sonarsource.com/Distribution/sonar-scanner-cli/sonar-scanner-cli-$SONAR_SCANNER_VERSION-linux.zip
+    - unzip -o $HOME/.sonar/sonar-scanner.zip -d $HOME/.sonar/
+    - export PATH=$SONAR_SCANNER_HOME/bin:$PATH
+
 tooling_tests_task:
   gke_container:
     <<: *CONTAINER_DEFINITION
     dockerfile: ci/Dockerfile
     cpu: 1
-    memory: 1G
+    memory: 2G
   env:
     PYTHONPATH: .
   install_dependencies_script:
     - cd rspec-tools
     - pipenv install
-    - pipenv run pip install pytest
+    - pipenv run pip install pytest-cov
```

```suggestion
    - pipenv run pip install pytest pytest-cov
```
Just want to make it explicit that the script needs both packages.